token/services/identity/role/role.go

```diff
@@ -21,8 +21,8 @@ import (
 type localMembership interface {
 	DefaultNetworkIdentity() driver.Identity
 	IsMe(ctx context.Context, id driver.Identity) bool
-	GetIdentityInfo(label string, auditInfo []byte) (idriver.IdentityInfo, error)
-	GetIdentifier(id driver.Identity) (string, error)
+	GetIdentityInfo(ctx context.Context, label string, auditInfo []byte) (idriver.IdentityInfo, error)
+	GetIdentifier(ctx context.Context, id driver.Identity) (string, error)
 	GetDefaultIdentifier() string
```

Comment:
should `GetDefaultIdentifier()` also receive a `ctx` to be consistent with `GetIdentifyInfo()`?